src/oled_display_node.cpp

```diff
@@ -190,118 +204,122 @@ static uint8_t sh1106_init_bytes[SH1106_INIT_BYTE_COUNT] = {
 };
 
 /*
- * @name                dispOled_writeBytes
- *
- * @name                display_setCursor
- * @brief               Move cursor to a given horizontal column and line
- *
- * @param               dispCtx         Context for display that holds type and hardware interface info
- * @param               column          The pixel resolution column from 0 to 127
- * @param               line            The line number where 0 is top line
+ * @name                i2c_read
+ * @brief               Read one or more bytes from an I2C based device
  *
- * @return              Returns 0 for ok or -1 for IO error
+ * @param               i2cDevFile      Name of the I2C device
+ * @param               i2c7bitAddr     7-bit I2C bus address
+ * @param               pBuffer         User 8-bit buffer for data return
+ * @param               numBytes        Number of bytes to be read from the chip
```

Comment:
Thank you for understanding and refactoring to avoid higher level context in low level driver.